containers/pipeline-agent/Pipfile

```diff
@@ -9,7 +9,7 @@ sphinx_rtd_theme = "==0.4.1"
 sphinx-autobuild = "==0.7.1"
 recommonmark = "==0.4.0"
 twine = "==1.11.0"
-ansible = "==2.6.4"
```

Comment:
I guess we can update it to 2.7

Reply:
Why?

Reply:
we use ansible 2.7.2 for playbooks installation, you can check in ansible container. So maybe that make sense to use the same version across all components, by the way it's not strictly necessary so up to you.